apps/src/templates/instructions/codeReview/ReviewNavigator.jsx

```diff
@@ -19,6 +21,25 @@ class ReviewNavigator extends Component {
     loadInProgress: false
   };
 
+  onSelectPeer = peer => {
```

Comment:
I'm kind of wondering if we should just have a `/instructions/codeReviewV2` directory and just copy/re-implement everything we need there.  This way we're guaranteed not to affect the current experience and it will be easier to figure out what to remove when we're done.

It seems like this component could be simplified if it were a functional component.  (I'm even wondering if this needs to be a separate component.)

Up to you!

Reply:
I think that's a good point. I'll make a codeReviewV2, but not as part of this PR. I'll follow-up with that